docs/reference/fpp-json-dict.md

```diff
@@ -453,6 +488,10 @@ Example JSON of an array of type U32 consisting of 10 elements
 [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
 ```
 
+### Constant Values
+Constant values include [Primitive Integer](#primitive-integer-values), [Floating-Point](#floating-point-values), 
+[String](#string-values), and [Boolean](#boolean-values) values.
```

Comment:
Should the spec call out which specific constants are pulled into the dictionary? Or maybe just the rationale on why some are and others aren't?

Reply:
I was thinking of adding that to the FPP user guide under the fpp-to-dict tool section.

Reply:
Makes sense to me!